### ISSUE TITLE
Demand projections

### ISSUE THREAD

**Issue by abhishek0208** [OSeMOSYS/osemosys_global#17]
Electricity demand by node (`SpecifiedAnnualDemand`) is currently constant. Future electricity demand will be projected based on factors such as population, GDP/capita, and urbanisation.

**Comment by maartenbrinkerink** [comment]
Working on it as we speak, will come with an update on Thursday.

**Comment by maartenbrinkerink** [comment]
I've updated OPG_demand_projection which for now is something we can work with. That said there are still plenty of aspects that need to be reassessed;

- Urbanization as variable (vs electricity demand) shows in a lot of cases to be exponential rather than linear.
- Electrification and decoupling not taken into account in demand forecast.
- Electricity demand increases linearly but at some point you expect saturation to occur even if GDP would grow further, not taken into account currently.

**Comment by maartenbrinkerink** [comment]
I suggest to close this issue for the time being.

**Comment by maartenbrinkerink** [comment]
I've made some updates in the demand projection externally and will include these in the coming week(s) within the OPG_demand_projection.py script.

**Comment by maartenbrinkerink** [comment]
Demand projections for the standard nodes are fully functional. Users have the ability to add in their own load data if required. There are still improvements to be made in the demand projections but is not a priority at the moment. Closing this issue accordingly.

**Comment by trevorb1** [comment]
@maartenbrinkerink did you run the workflow to confirm that demand projections were working? Recently (within the past couple weeks) @SimoneOseiOwusu and I ran into an issue where the world bank API calls were failing through the `world_bank_data` package. The issue ended up being that the API call is using `http` rather than `https`. We manually fixed it in our case. Did you run inot this same issue? 

I agree, it does not belong in this issue ticket, just curious if you had the same issue?
